day_1_ddo/ddo_hackathon/admin/app.py

```diff
@@ -161,21 +176,32 @@ def main():
 
                 with status_container:
                     if "All tests passed successfully!" in test_results:
-                        valid_save_path = save_valid_submission(code_input, team_name)
+                        valid_save_path = save_valid_submission(code_input, team_name, track)
                         st.success(
                             f"{test_results}\n\nSubmission saved to {valid_save_path}."
                         )
                     else:
                         st.error(test_results)
 
-    # Leaderboard Tab
+    # Leaderboard Tab for Track 1
     with tabs[1]:
-        st.title("Leaderboard")
+        st.title("Leaderboard - Track 1")
+        try:
+            # Assuming your Track 1 leaderboard HTML file is named "leaderboard_t1.html"
+            leaderboard_html = get_leaderboard_html(file_name="day1/leaderboard_t1.html")
+            st.components.v1.html(leaderboard_html, height=600, scrolling=True)
+        except Exception as e:
+            st.error("No leaderboard generated yet!")
+
+    # Leaderboard Tab for Track 2
+    with tabs[2]:
+        st.title("Leaderboard - Track 2")
         try:
-            leaderboard_html = get_leaderboard_html()
+            # Assuming your Track 2 leaderboard HTML file is named "leaderboard_t2.html"
+            leaderboard_html = get_leaderboard_html(file_name="day1/leaderboard_t2.html")
             st.components.v1.html(leaderboard_html, height=600, scrolling=True)
         except Exception as e:
-            st.error(f"Could not load the leaderboard: {e}")
+            st.error("No leaderboard generated yet!")
     
 if __name__ == "__main__":
-    main()
\ No newline at end of file
+    main()
```

Comment:
```suggestion
    main()
    
```

Reply:
This is not a big one, but maybe `black` did not work here? I think the trailing empty line is standard to have afaik.